gradoop-flink/src/main/java/org/gradoop/flink/io/impl/rdbms/connection/FlinkDatabaseInputHelper.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.io.impl.rdbms.connection;
+
+import org.apache.flink.api.java.DataSet;
+import org.apache.flink.api.java.ExecutionEnvironment;
+import org.apache.flink.api.java.io.jdbc.JDBCInputFormat;
+import org.apache.flink.api.java.io.jdbc.split.GenericParameterValuesProvider;
+import org.apache.flink.api.java.typeutils.RowTypeInfo;
+import org.apache.flink.types.Row;
+
+/**
+ * Queries the relational data
+ */
+public class FlinkDatabaseInputHelper {
+
+  /**
+   * Provides an flink input to a relational database and querying data in a
+   * distributed manner.
+   *
+   * @param env
+   *          Flink Execution Environment
+   * @param rdbmsConfig
+   *          Configuration of the used database management system
+   * @param rowCount
+   *          Number of table rows
+   * @param sqlQuery
+   *          Valid sql query
+   * @param typeInfo
+   *          Database row type information
+   * @return DataSet of type row, consisting of queried relational data
+   * @throws ClassNotFoundException
```

Comment:
Isn't that a `RuntimeException`? If so: remove the throws clause, if not: Add javadoc comment.